src/components/PageDefinitions.js

```diff
@@ -212,7 +212,7 @@ export class PageDefinitions extends AbstractPageDefinitions {
         <MenuItem eventKey="1" onSelect={this.doSaveAsUrl}>
           URL
         </MenuItem>
-        <MenuItem eventKey="2" onSelect={this.doSave}>
+        <MenuItem eventKey="2" onSelect={() => this.setState({ visible: true })}>
```

Comment:
`visible` is a pretty generic name for the "pick a filename popup". Perhaps `savePopupVisible` or some such?